### DISCUSSION TITLE
ENH: Doc: DTI format

### DISCUSSION THREAD

**Discussion by EmmaRenauld** [dipy/dipy#2675] [Reconstruction] [answered]
Hi all,

Even after many researches both on Google, in the doc or directly on Github, I can't find a description of the expected file format when saving a tensor.

Ex: 
mrtrix creates a 4D file with [D11, D22, D33, D12, D13, D23]
FSL creates a 4D file with [Dxx, Dxy, Dxz, Dyy, Dyz, Dzz]
ANTS creates a 5D file with [dxx, dxy, dyy, dxz, dyz, dzz].

What is your format? How can we obtain something similar? The tutorial here https://dipy.org/documentation/1.5.0/examples_built/reconst_dti/#example-reconst-dti should mention it. The code in `dipy.reconst.dti` has poor documentation of the formats used. In particular, method `lower_triangle`  should be clearer. 

If I'm not mistaken, based on `_lt_rows, _lt_cols` you are creating [Dxx, Dxx, Dyy, Dxz, Dyz, Dzz], is that right?

**Comment by skoudoro** [comment, marked as answer]
Hi @EmmaRenauld,

Yes, This is  [Dxx, Dxx, Dyy, Dxz, Dyz, Dzz],

You can see it in the code:
- https://github.com/dipy/dipy/blob/master/dipy/reconst/dti.py#L1639
- https://github.com/dipy/dipy/blob/master/dipy/workflows/reconst.py#L304
- https://github.com/dipy/dipy/blob/master/dipy/reconst/tests/test_dti.py#L325

I agree that we need to improve the documentation concerning this point.

Documentation contributions are also welcomed! I would be great if you could create a PR concerning this point and update the tutorial.

Thanks for the feedback.
